contributions/catalog/webapps/static/css/style.css

```diff
@@ -78,24 +79,94 @@ html * {
     clip-path: polygon(0 50%, 100% 0, 100% 100%, 0% 100%);
 }
 
-input[type=text], select, textarea {
-    color: #090000;
+label {
+    font-size: 16px;
+    font-weight: 500;
+    font-stretch: normal;
+    font-style: normal;
+    letter-spacing: normal;
+    height: 20px;
+    color: #13294b;
+}
+
+input[type=text], input[type=url], select, textarea {
+    color: #404040;
```

Comment:
```
.Rectangle-Copy-6 {
  width: 288px;
  height: 48px;
  padding: 12px 8px 16px;
  border: solid 1px #002855;
  background-color: #fff;
}

input {
  width: 272px;
  height: 20px;
  font-family: ProximaNova;
  font-size: 16px;
  font-weight: normal;
  font-stretch: normal;
  font-style: normal;
  line-height: 1.25;
  letter-spacing: normal;
  color: #404040;
}
```

Copying the textbox style and hint text from Zeplin. 

Please remove the rounded edges currently present, and fix the height and style of the hint text style (if possible).

Reply:
updated box styling and placeholder fonts

Reply:
Thanks.